amulet/level/formats/leveldb_world/format.py

```diff
@@ -310,3 +311,37 @@ def _get_raw_chunk_data(
         :return: The raw chunk data.
         """
         return self._level_manager.get_chunk_data(cx, cz, dimension)
+
+    def get_players(self) -> Generator[str, None, None]:
+        """
+        Returns a generator of all player ids that are present in the level
+        """
+        yield from (
+            pid[7:].decode("utf-8")
+            for pid, _ in self._level_manager._db.iterate(b"player_", b"player_\xFF")
+        )
+
+    def get_player(self, player_id: str = LOCAL_PLAYER) -> Player:
+        """
+        Gets the :class:`Player` object that belongs to the specified player id
+
+        If no parameter is supplied, the data of the local player will be returned
+
+        :param player_id: The desired player id
+        :return: A Player instance
+        """
+        if player_id == LOCAL_PLAYER:
+            key = player_id
+        else:
+            key = f"player_{player_id}".encode("utf-8")
+        try:
+            data = self._level_manager._db.get(key)
+        except KeyError:
+            raise KeyError(f"Player {player_id} doesn't exist")
+        player_nbt = nbt.load(buffer=data, compressed=False, little_endian=True)
```

Comment:
The buffer key word argument has been removed from the nbt load function.
The first input can be either a string path to a file to load, a file-like object (with a read method) or a bytes object of the actual nbt data. It should look like this
```
nbt.load(data, compressed=False, little_endian=True)
```